### PR TITLE
Refactor external links

### PR DESCRIPTION
**Pull request by banukutlu**
refactoring for #705
Issue: #672

just a bit reorg for external links @nicktiberi could you pls review/check? what do you think?

### CODE REVIEW

File: spec/models/access_online_links_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-RSpec.describe AccessOnlineLinks do
+RSpec.describe ExternalLinks do
```
Comment (nicktiberi) [inline spec/models/access_online_links_spec.rb:5]: Maybe we should rename this file to `external_links_spec.rb` to match the new name of the spec?

File: spec/models/access_online_links_spec.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
   describe '#access_online_links' do
```
Comment (nicktiberi) [inline spec/models/access_online_links_spec.rb:6]: Probably rename this as well to `#external_links`?

File: app/views/external_links/_all.html.erb [hunk 3]
```diff
@@ -1,5 +1,8 @@
-<%= render partial: 'external_links/psu_digital_collections_links', locals: { psu_digital_collections_links: document.psu_digital_collections_links } if document.psu_digital_collections_links %>
+<% psu_digital_collections_links = document.psu_digital_collections_links
+   access_online_links = document.access_online_links %>
```
Comment (nicktiberi) [inline app/views/external_links/_all.html.erb:2]: Do we want to add `hathi_links` to this list of variables, and refactor the partial declaration similarly?

File: app/models/external_links.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ExternalLinks
+  def psu_digital_collections_links
```
Comment (nicktiberi) [inline app/models/external_links.rb:4]: Looking much cleaner! 😎 👍

**Comment by nicktiberi** [comment]
Thanks for cleaning this up, @banukutlu! 🙂

File: spec/models/access_online_links_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/models/access_online_links_spec.rb:5]: oh yes! totally missed this one :) good catch :)

File: spec/models/access_online_links_spec.rb (hunk 2 above)
Comment (banukutlu) [reply spec/models/access_online_links_spec.rb:6]: yup

File: spec/models/access_online_links_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/models/access_online_links_spec.rb:5]: ok just changed with the latest commit

File: app/models/external_links.rb (hunk 4 above)
Comment (banukutlu) [reply app/models/external_links.rb:4]: 😎 👍

File: app/views/external_links/_all.html.erb (hunk 3 above)
Comment (banukutlu) [reply app/views/external_links/_all.html.erb:2]: yeah good idea 👍

Comment (banukutlu) [reply app/views/external_links/_all.html.erb]: this is done with the latest commit, thanks 👍

**Pull request merged**